pkg/api/licenses.go

```diff
@@ -884,3 +884,34 @@ func SearchInLicense(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, res)
 }
+
+// ExportLicenses gives users all licenses as a json file.
+//
+//	@Summary		Export all licenses as a json file
+//	@Description	Export all licenses as a json file
+//	@Id				ExportLicenses
+//	@Tags			Licenses
+//	@Produce		json
+//	@Success		200	{array}		models.LicenseJson
```

Comment:
Oops, a minor change needed.
```suggestion
//	@Success		200	{array}		models.ExportLicenseDB
```

Reply:
@Kaushl2208 , Can you please change the file name with following comment as well?

https://github.com/fossology/LicenseDb/pull/53#issuecomment-2109355660